scripts/cat/status.py

```diff
@@ -175,10 +167,9 @@ def _start_group_history(
         new_history = {"group": group, "rank": rank, "moons_as": 0}
 
         if not age and not rank:
-            print(
+            raise Exception(
```

Comment:
A more precise exception would be better here.
```suggestion
            raise ValueError(
```